frontend/src/components/common/AppLayout/AppLayout.tsx

```diff
@@ -35,17 +38,36 @@ const AppLayout: FC<IAppLayoutProps> = ({ ...props }) => {
         />
         <Content className="flex">
           <Switch>
-            {routes.map(r =>
-              r.content ? (
-                <Route exact key={r.route.path} path={r.route.path}>
-                  <Row className="h-full pt-5 xs:pt-10 pb-20 flex w-full px-4">
-                    <Col span={0} lg={1} xxl={2}></Col>
-                    {r.content}
-                    <Col span={0} lg={1} xxl={2}></Col>
-                  </Row>
-                </Route>
-              ) : null
+            {hasRenderingError(errorsQueue) ? (
+              <div className="flex justify-center w-full items-center">
+                <Result
+                  status="500"
+                  title="Application error."
+                  subTitle={'Sorry, something went wrong.'}
+                />
```

Comment:
Is it possible to add the refresh button here?
(also I would add a comment after the `status="500"`, saying it's like that to show a certain image, since this is clearly not a 500 error, being client-side)

Reply:
If you prefer i can add a button to clear the error queue, so the app can rerender the routes

Reply:
The queue is not meant to be cleared. 
I said refresh button but it could be enough to point to the `/`, so that we get back to a theoretically _working_ place. However if we don't put the refresh button I'd also add a `If this keeps happening, try refreshing the page.`

Reply:
If you try to redirect to `/` you still see the same page.

Reply:
I've changed some things, please check it again.